roles/kafka_connect/tasks/connect_plugins.yml

```diff
@@ -10,13 +10,11 @@
   with_items: "{{ kafka_connect_final_properties['plugin.path'].split(',') }}"
 
 - name: Installing Local Plugins
-  unarchive:
-    src: "{{item}}"
-    dest: "{{kafka_connect_plugins_dest}}"
-    group: "{{kafka_connect_group}}"
-    owner: "{{kafka_connect_user}}"
-    mode: 0755
-    remote_src: false
+  shell: |
+    {{ (installation_method == 'archive') | ternary(archive_destination_path + '/confluent-' + confluent_package_version, '/usr') }}/bin/confluent-hub install \
```

Comment:
Why don't we use `binary_base_path` instead

Reply:
done, thank you @nsharma-git ! 👍